zproject/settings.py

```diff
@@ -159,6 +159,8 @@ def get_config(section: str, key: str, default_value: Optional[Any]=None) -> Opt
     'SOCIAL_AUTH_GITHUB_TEAM_ID': None,
     'SOCIAL_AUTH_SUBDOMAIN': None,
     'SOCIAL_AUTH_AZUREAD_OAUTH2_SECRET': get_secret('azure_oauth2_secret'),
+    'SOCIAL_AUTH_GOOGLE_KEY': None,
+    'SOCIAL_AUTH_GOOGLE_SECRET': get_secret('social_auth_google_secret', development_only=True),
```

Comment:
This logic isn't quite right -- it's the `key` part that could reasonable have `development_only` on it.  Fixed and pushed back to your PR